src/eventbuffer/filterAttributes.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ *
+ * @param attributes received with the reported event.
+ * @param attributeToFilter set of attributes like 'externalMeetingId', 'externalUserId' to filter.
+ * @returns filtered attributes.
+ */
+const filterAttributes = (
```

Comment:
Does this need its own routine like this? If so, this seems like a helper function so should we rename it to eventBufferUtil.ts or something?

Reply:
Yes, it was separated so that we can test the function separately; add unit tests just for it. It is under `eventbuffer` so I think the name should be fine and the parameters make is easy to use more generally hence the name is general. For now, it has restricted types which can be expanded to support other more complex types in future. So should I rename it then?

Reply:
It seems like it is used in only one place right now so its strange to separate it out just for testing. It is not useful to use in other places anyway since it is nested in event buffer.  It also increases the public API unnecessarily.

Reply:
Ok separated out as a private function in the buffer itself.